pyneuroml/neuron/analysis/HHanalyse.py

```diff
@@ -44,6 +44,13 @@ def process_args():
         help="Supress plotting of variables and only save to file",
     )
 
+    parser.add_argument(
+        "-save_plots",
```

Comment:
let's use `savePlots` here to keep in line with the already used camelCase style

Reply:
Done